tensorflow/c/experimental/ops/math_ops.cc

```diff
@@ -40,14 +40,14 @@ Status Mul(AbstractContext* ctx, absl::Span<AbstractTensorHandle* const> inputs,
 Status Conj(AbstractContext* ctx,
             absl::Span<AbstractTensorHandle* const> inputs,
             absl::Span<AbstractTensorHandle*> outputs, const char* name) {
-  auto dtype = inputs[0]->DataType();
-  if (DataTypeIsFloating(BaseType(dtype)) ||
-      DataTypeIsInteger(BaseType(dtype))) {
-    TF_RETURN_IF_ERROR(Identity(ctx, inputs, outputs, name));
-  } else {
-    return errors::Unimplemented("Conj does not support complex types yet.");
-  }
-  return Status::OK();
+  AbstractOperationPtr conj_op(ctx->CreateOperation());
```

Comment:
Could we move your code to the else branch of the previous code? For non-complex types `Conj` is a no-op.

Reply:
Done